sdks/python/apache_beam/io/range_trackers.py

```diff
@@ -285,6 +354,17 @@ def fraction_consumed(self):
                        ' consumed due to positions being opaque strings'
                        ' that are interpreted by the service')
 
+  def split_points(self):
+    with self._lock:
```

Comment:
This is for my information. What is the need for locks in this files? Is it being accessed by multiple threads?